vme/src/pcsave.cpp

```diff
@@ -52,16 +55,21 @@ void assign_player_file_index(unit_data *pc)
     }
 }
 
-char *PlayerFileName(const char *pName)
+std::string PlayerFileName(const char *pName)
 {
-    static char Buf[MAX_INPUT_LENGTH + 1 + 512];
-    char TmpBuf[MAX_INPUT_LENGTH + 1];
+    std::string TmpBuf;
 
-    strcpy(TmpBuf, pName);
-    str_lower(TmpBuf);
-    snprintf(Buf, sizeof(Buf), "%s%c/%s", g_cServerConfig.getPlyDir().c_str(), *TmpBuf, TmpBuf);
```

Comment:
So I changed this, but not 100% that I am correct - see here there is only 1 / in the string - this seems like the lib/ply directory with the hash buckets of 1 char of username being the top level directory